docs/tutorial/part-two/index.md

```diff
@@ -116,6 +116,7 @@ Similar to part one, open a new terminal window and run the following to create
 
 ```shell
 gatsby new tutorial-part-two https://github.com/gatsbyjs/gatsby-starter-hello-world
+cd tutorial-part-two
```

Comment:
And a similar change here.

Reply:
When going through a tutorial, if there are commands that need to be run, I think it is clearer to have those commands all shown in the same context. Having a mix of block and inline quotes makes it easier (imo) to miss a step. Changing into the directory immediately after the `gatsby new` command makes it clear that all subsequent commands need to be entered from within that directory, and all files to be edited use that directory as the root.

I can make the change but I believe it will be less clear to newcomers. Perhaps others can chime in here with their thoughts.

Reply:
I agree with @hazelnusse that each command ought to be formatted the same so that people don't miss them. Another option would be to explain in tutorial part one that it's necessary to immediately cd into the project directory before proceeding to work on it and then just expect everyone to remember to do that without the tutorial reminding them. It depends on what kind of audience we're writing for--total beginners or intermediate/advanced?

Reply:
In my case, I had never used node.js or npm until going through the tutorial so for me, having an explicit reminder to go into the directory before running the npm install command is preferable. For an intermediate/advanced user who already knows this, I don't see much downside in 1 extra line so I'd suggest going with the more explicit option on each tutorial.

Reply:
Ok, that makes sense! Let's do it. My only last question is whether the format will make running these commands confusing. Usually, the `gatsby new tutorial-part-one https://.........` is so long, it runs past it's container box and so people who are trying to copy and paste it might accidentally copy the next line `cd tutorial-part-one`? That's my concern. Do you think that could happen? If so, we could either adjust the container so it doesn't allow horizontal scrolling and creates line breaks automatically so the command fits (@fk what do you think of this?), OR put `cd tutorial-part-one` or any `cd` command right before the next command they run. Does that make sense?

Reply:
Or just merge these changes and see if anyone complains ;) that's an option, too haha!

Reply:
Yeah, repeating every command people need to run totally makes sense to me 👍

Reply:
@shannonbux copying both is fine — when you paste multiple lines into the terminal, it'll run each line in turn.

Reply:
@KyleAMathews @shannonbux is there anything else you need before this is good to merge? I'm fine with it as it is.